### DISCUSSION TITLE
The machine learning models have yet to be downloaded

### DISCUSSION THREAD

**Discussion by backupandforkacc** [nextcloud/recognize#853] [General]
I installed the nextcloud app yesterday in the evening and it said:
"The machine learning models have yet to be downloaded"

This morning, still got the same problem, even after clearing cache or browser. Here is the full screenshot:
![grafik](https://github.com/nextcloud/recognize/assets/1709199/75d46d52-f048-4d26-b1be-efe0db406d9e)

To me, the documentation is also not clear in one point: Where do the Info about the name of the person come from? Is is taken from "contacts"?

**Comment by marcelklehr** [comment]
Hello

> Where do the Info about the name of the person come from? Is is taken from "contacts"?

This was the case in version 2, but since version 3 we do not automatically name clusters anymore.

> "The machine learning models have yet to be downloaded"

Mh, it seems to work although the models are not downloaded, that's a first. In any case you can download the models on the command line using occ recognize:download-models.
